app/models/manageiq/providers/openstack/network_manager/network_router.rb

```diff
@@ -2,10 +2,26 @@ class ManageIQ::Providers::Openstack::NetworkManager::NetworkRouter < ::NetworkR
   include ProviderObjectMixin
   include AsyncDeleteMixin
 
-  supports :create_network_router
-  supports :delete_network_router
-  supports :update_network_router
```

Comment:
@gildub @tzumainn do you see a way of removing these features from the SupportsFeatureMixin and also from NetworkManager?